app/src/main/java/me/darthwithap/hotel_app/presentation/reservation/success/ReservationSuccessScreen.kt

```diff
@@ -0,0 +1,401 @@
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.unit.dp
+import cafe.adriel.voyager.core.screen.Screen
+import me.darthwithap.hotel_app.R
+import me.darthwithap.hotel_app.ui.components.ButtonSize
+import me.darthwithap.hotel_app.ui.components.SecondaryButton
+import me.darthwithap.hotel_app.ui.theme.AppTheme
+import java.util.UUID
+
+object ReservationSuccessScreen : Screen {
+    @Composable
+    override fun Content() {
+        ReservationSuccessScreen()
+    }
+}
+
+@Composable
+fun ReservationSuccessScreen(
+    modifier: Modifier = Modifier
+) {
+    LazyColumn(
+        modifier = modifier
+            .systemBarsPadding()
+            .background(color = AppTheme.colors.lightCard),
+    ) {
+        item {
+            Spacer(modifier = Modifier.height(16.dp))
+        }
+
+        item {
+            ReservationTitle(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 20.dp)
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(24.dp))
+        }
+
+        item {
+            InvoiceCard()
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(24.dp))
+        }
+
+        item {
+            FlightRecommendationCard()
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(24.dp))
+        }
+
+        item {
+            BackToHomeButton(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 24.dp),
+                onClick = {}
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(24.dp))
+        }
+    }
+}
+
+@Composable
+private fun ReservationTitle(
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text(
+            text = stringResource(R.string.reservation_success),
+            style = AppTheme.typography.headlineMedium26Bold.copy(fontWeight = FontWeight.Normal),
+            color = AppTheme.colors.black100,
+        )
+
+        Spacer(modifier = Modifier.height(8.dp))
+
+        Text(
+            modifier = Modifier.padding(horizontal = 52.dp),
+            textAlign = TextAlign.Center,
+            text = buildAnnotatedString {
+                withStyle(
+                    style = AppTheme.typography.forms16Regular.toSpanStyle()
```

Comment:
Here, in the figma file, title14 typography is used, please check

Reply:
@patthhar I have changed the font size according to Figma. Please have a look again.